### PR TITLE
Beehive upgrade for multiuser threading

### PR DESCRIPTION
**Pull request by BaIthamel**
Added a -t flag to designate number of threads per worker
## Description

when using the -t flag the script then generates a bat and .sh that has multiple user credentials per worker line
## Motivation and Context

this simplifies beehive generation now that throttling has been increased
## How Has This Been Tested?

limited testing on linux, more appreciated.
One user, repeated use, no issues  on windows.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Tools/Hex-Beehive-Generator/location_generator.py [hunk 1]
```diff
@@ -35,13 +36,13 @@
     branchpath = args.installdir
     executable = args.installdir + "\\runserver.py"
     auth_template = '-a {} -u {} -p "{}"'  # windows people want double-quoted passwords
-    actual_worker_params = '{auth} -ns -l "{lat}, {lon}" -st {steps}'
-    worker_template = 'Start "{{threadname}}" /d {branchpath} /MIN {pythonpath} {executable} {actual_params}\nping 127.0.0.1 -n 6 > nul\n\n'.format(
+    actual_worker_params = '{auth}-ns -l "{lat}, {lon}" -st {steps}'
```
Comment (Talamar1) [inline Tools/Hex-Beehive-Generator/location_generator.py]: I believe you will need a space after {auth}

Comment (BaIthamel) [inline Tools/Hex-Beehive-Generator/location_generator.py]: Nope that's been moved to auth template to allow for concatenating those strings on themselves. Oh fk windows has it's own auth 1 sec

**Comment by Likw3d** [comment]
Im Getting 

File "location_generator.py", line 139

SyntaxError: invalid syntax

When trying to run anything even help.

**Comment by BaIthamel** [comment]
It's only 138 lines long what the hell?

**Comment by reyqn** [comment]
You are missing a parenthesis at the last line.

**Comment by skahandz** [comment]
138: +        print("{}, {}".format(location.lat, location.lon)

to 

138:+        print("{}, {}".format(location.lat, location.lon))

**Comment by BaIthamel** [comment]
Okay fixed that.

**Comment by rayuki** [comment]
tested the latest over multiple areas, with up to 1000 accounts and it works great no matter what -t i throw at it.

**Comment by JZ6** [comment]
duplicate of #3137

**Comment by JZ6** [comment]
@Ashex

**Comment by Ashex** [comment]
I'd rather we got this merged in first, I'll rebase then incorporate the changes into my refactor. I would like to then get that PR merged before other beehive PRs since a simple rebase isn't possible.

**Pull request closed without merging**